### PR TITLE
Update event-views.md

### PR DESCRIPTION
**Pull request by msbemba**
Using site-relative links as the image is from a different repo

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5014

File: microsoft-365/security/defender-endpoint/event-views.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ You can also manually navigate to the event area that corresponds to the feature
 
 3. Select **Action** > **Import Custom View...**
 
-    ![Animation highlighting Import custom view on the left of the Even viewer window](/windows/security/threat-protection/images/events-import)
+    ![Animation highlighting Import custom view on the left of the Even viewer window](/windows-itpro-docs/windows/security/threat-protection/images/events-import.gif)
 
 4. Navigate to where you extracted XML file for the custom view you want and select it.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/event-views.md]:
```suggestion
4. Navigate to where you extracted the XML file for the custom view you want and select it.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - one minor additional edit, please.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@Dansimp Gentle reminder, please merge this PR. Thanks

**Comment by Dansimp** [comment]
I don't think referencing the name of the repo will work here. I will need to double check. we might need to move the image files over to this repo. I will investigate.

**Comment by yogkumgit** [comment]
@msbemba and @Dansimp

**Comment by yogkumgit** [comment]
@Dansimp Gentle reminder, please merge or reply. Awaiting resolution. Thanks

**Pull request merged**